tests/python/PycodestyleBearTest.py

```diff
@@ -63,3 +69,22 @@ def hello():
     valid_files=(file_with_very_long_line,),
     invalid_files=(),
     settings={'max_line_length': 0})
+
+PycodestyleBearAspectsTest = verify_local_bear(
```

Comment:
We need to verify that the `Result` object returned uses the aspect.
(I suspect that it doesnt.)

If it doesnt, create a bug against `@linter` because its default `run` needs to use aspects for the `Result`, and `map_setting_to_aspect` may also need to be improved in the process.

And we may just merge this as-is if the bug is in Linter.

If the Result objects correctly use aspects, then this test class needs to be a more detailed test class ... which checks that the Result object reports the correct aspect.

Reply:
Yes, it doesn't use the aspect. 

We need to modify `@linter` to use the aspect just as the `Result` module does.

Reply:
Ya, I expected so.  Thanks.

What you could try is add a `run` method to this bear, and it would do

```
for result in super().run(*args, **kwargs):
   if (max_line_length is from aspect) and (result is line length problem):
        result.aspect = LineLength('Python').max_line_length
```

Reply:
Now we need a test class which verifies that the aspect is used when the line length is exceeded.

Reply:
> Ya, I expected so. Thanks.
> 
> What you could try is add a `run` method to this bear, and it would do
> 
> ```
> for result in super().run(*args, **kwargs):
>    if (max_line_length is from aspect) and (result is line length problem):
>         result.aspect = LineLength('Python').max_line_length
> ```
I don't think so, defining such if-else for each aspect will be a nice idea. There will be a lot of if-else then because there exists many leaf-aspects for which we will have to define a rule then.
Can we use nltk package? that can somewhere help us to extract meaningful words from result message and then check whether that result belongs to that leaf-aspect or not?

Reply:
> Yes, it doesn't use the aspect.
> We need to modify `@linter` to use the aspect just as the `Result` module does.
> 
> I have created an issue [coala/coala#5694](https://github.com/coala/coala/issues/5694)

Do you mean here by LocalBear and GlobalBear which are having a `aspects` argument? And something like that we need to implement for `@linter` also?